.unreleased/pr_6767

```diff
@@ -0,0 +1,2 @@
+Implements: #6767 Add metadata table `_timestaledb_internal.bgw_job_stat_history` for tracking job execution history
+Thanks: @mahipv For starting working on this PR
```

Comment:
Since the _Implements_ and _Thanks_ lines are part of different sections in the changelog:

```suggestion
Thanks: @mahipv For starting working on the job history PR
```

Reply:
Fixed!